test/tools/libtesteth/TestHelper.cpp

```diff
@@ -271,45 +271,44 @@ std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _di
 json_spirit::mValue convertYamlNodeToJson(YAML::Node _node)
 {
 	if (_node.IsNull())
-		return json_spirit::mObject();
+		return json_spirit::mValue();
 
 	if (_node.IsScalar())
 	{
-		json_spirit::mValue value;
 		if (_node.Tag() == "tag:yaml.org,2002:int")
-			value = _node.as<int>();
+			return _node.as<int>();
 		else
-			value = _node.as<std::string>();
-		return value;
+			return _node.as<string>();
 	}
 
 	if (_node.IsMap())
 	{
 		json_spirit::mObject jObject;
 		for (auto const& i : _node)
-			jObject[i.first.as<std::string>()] = convertYamlNodeToJson(i.second);
+			jObject.emplace(i.first.as<string>(), convertYamlNodeToJson(i.second));
 		return jObject;
 	}
 
 	if (_node.IsSequence())
 	{
 		json_spirit::mArray jArray;
-		for (std::size_t i = 0; i < _node.size(); i++)
-			jArray.push_back(convertYamlNodeToJson(_node[i]));
+		for (size_t i = 0; i < _node.size(); i++)
+			jArray.emplace_back(convertYamlNodeToJson(_node[i]));
 		return jArray;
 	}
 
+	BOOST_ERROR("Error parsing YAML node. Element type note defined!");
 	return json_spirit::mValue();
 }
 
-// this function is here so not to include YAML into other files
-json_spirit::mValue parseYamlToJson(std::string const& _string)
+/// this function is here so not to include YAML into other files
```

Comment:
I don't understand this phrase: "Include YAML into other files".

* including YAML in JSON files?  Why would that happen?
* including YAML in CPP sources?  Maybe not.

Reply:
so not to include yaml headers in other cpp files in project cpp-ethereum/test